packages/shared/src/common/enums/api/routes.ts

```diff
@@ -36,6 +37,7 @@ export enum ProductApiRoutes {
   FAVORITE = '/favorite',
   FAVORITE_IDS = '/favorite-ids',
   BUY = '/buy/:id',
+  CURRENT_PRICE = '/current-price',
```

Comment:
you don't need this route.

Api call place a bid could return the product dto with new price